src/main/scala/cromwell/engine/WorkflowManager.scala

```diff
@@ -47,50 +65,79 @@ trait WorkflowManager {
 /**
  * A WorkflowManager using WorkflowActors as the workflow handler
  */
-trait ActorWorkflowManager extends WorkflowManager {
-  override type Workflow = ActorRef // TODO: In a world where Akka Typed is no longer experimental switch to that
+
+object ActorWorkflowManager {
+  def props: Props = Props(classOf[ActorWorkflowManager])
+}
+
+class ActorWorkflowManager extends WorkflowManagerActor {
   override val backend = new LocalBackend
-  implicit def actorSystem: ActorSystem
+  implicit val timeout = Timeout(30.seconds)
 
-  override def generateWorkflow(id: WorkflowId, wdl: WdlSource, inputs: WorkflowInputs): Try[Workflow] = {
+  override def generateWorkflow(id: WorkflowId, wdl: WdlSource, inputs: binding.WorkflowInputs): Try[Workflow] = {
     val binding = Try(WdlBinding.process(wdl))
-    binding map {x => actorSystem.actorOf(WorkflowActor.buildWorkflowActorProps(id, x, inputs))}
+    binding map {x => context.actorOf(WorkflowActor.buildWorkflowActorProps(id, x, inputs))}
   }
 
-  override def workflowStatus(id: WorkflowId): Option[WorkflowState] = workflowById(id) map {x => workflowState(x)}
+  override def workflowOutputs(id: WorkflowId): Future[Option[binding.WorkflowOutputs]] = {
+    workflowById(id) match {
```

Comment:
Can this just be something like `Future { workflowById(id) map {...}  }`?  Taking apart the `Option` to put it back together feels a icky.